### PR TITLE
Shift to use new imported ifrau client

### PR DESCRIPTION
**Pull request by grant-cleary**
Bump the hello-world FRA to use the latest ifrau client so we can test this new workflow within an actual FRA.

### CODE REVIEW

**Comment by grant-cleary** [comment]
Note: This can't/shouldn't merge until the requisite ifrau version bump PRs are merged and released in core/htmleditor.

File: rollup.config.js [hunk 1]
```diff
@@ -26,6 +27,7 @@ export default merge(baseConfig, {
 		}),
 		dynamicImportVars({
 			include: 'node_modules/@brightspace-ui/htmleditor/**/*.js'
-		})
+		}),
+		nodeResolve()
```
Comment (grant-cleary) [inline rollup.config.js:31]: Required for rollup to actually resolve ifrau.

**Review by dlockhart — APPROVED** [review]

**Review by dbatiste — APPROVED** [review]

**Comment by grant-cleary** [comment]
New core/htmleditor versions have been published with the new ifrau usage, so I'll do a quick test run locally and then merge this if everything looks good.

**Comment by grant-cleary** [comment]
Ok, core and htmleditor changes have now been merged and released, so I'll merge this.

**Pull request merged**